LICENSE

```diff
@@ -1,339 +1,205 @@
                     GNU GENERAL PUBLIC LICENSE
                        Version 2, June 1991
 
- Copyright (C) 1989, 1991 Free Software Foundation, Inc.,
```

Comment:
You can actually delete this file, it's replaced by the one in the `LICENSES` directory.